examples/alphaTestExamples/MultipleServcies.md

```diff
@@ -0,0 +1,131 @@
+[kind]: https://github.com/kubernetes-sigs/kind
+
+# Demo: Multiple Services
+
+The following demonstrates applying and destroying multiple services to a `kind` cluster.
+
+Steps:
+1. Download the resources files for wordpress, mysql services.
+2. Spin-up kubernetes cluster on local using [kind].
+3. Deploy the wordpress, mysql services using kapply and verify the status.
+4. Destroy wordpress service and verify that only wordpress service is destroyed.
+
+First define a place to work:
+
+<!-- @makeWorkplace @testE2EAgainstLatestRelease -->
+```
+DEMO_HOME=$(mktemp -d)
+```
+
+Alternatively, use
+
+> ```
+> DEMO_HOME=~/hello
+> ```
+
+## Establish the base
+
+Download the example configs for services `mysql` and `wordpress`
+<!-- @createBase @testE2EAgainstLatestRelease -->
+```
+BASE=$DEMO_HOME/base
+mkdir -p $BASE
+OUTPUT=$DEMO_HOME/output
+mkdir -p $OUTPUT
+
+mkdir $BASE/wordpress
+mkdir $BASE/mysql
+
+curl -s -o "$BASE/wordpress/#1.yaml" "https://raw.githubusercontent.com\
+/kubernetes-sigs/kustomize\
+/master/examples/wordpress/wordpress\
+/{deployment,service}.yaml"
+
+curl -s -o "$BASE/mysql/#1.yaml" "https://raw.githubusercontent.com\
+/kubernetes-sigs/kustomize\
+/master/examples/wordpress/mysql\
+/{secret,deployment,service}.yaml"
+
+expectedOutputLine()
+{
+  test 1 == \
+  $(grep "$@" $OUTPUT/status | wc -l); \
+  echo $?
+
+}
```

Comment:
I'd tighten this up a bit since the user is expected to copy paste, i.e.
```
function blah {
// stuff
}
```
i.e. no blank lines, no `{` by itself on a line

Reply:
Sorry about that. Done